documentation/src/components/side-nav.ts

```diff
@@ -15,9 +15,11 @@ import {
     CSSResultArray,
     property,
     PropertyValues,
+    TemplateResult,
 } from 'lit-element';
 import { ComponentDocs } from '../../components';
```

Comment:
This uses different data than `import('../../components/searchIndex.json')` so the initial navigation listings are different than the search listings...